src/org/zaproxy/zap/view/table/DefaultHistoryReferencesTableModel.java

```diff
@@ -247,6 +247,18 @@ public void addHistoryReference(HistoryReference historyReference) {
     }
 
     /**
+     * Convenience method that removes a {@code DefaultHistoryReferencesTableEntry} with the given history reference and adds it
+     * from the model.
+     * 
+     * @param historyReference the history reference that will be removed from the model
+     * @see DefaultHistoryReferencesTableEntry
+     * @see HistoryReference
+     */
+    public void removeHistoryReference(HistoryReference historyReference) {
```

Comment:
- Please, add the since tag.
- Might worth document (or validate) that the `historyReference` should not be null.